server/src/test/java/org/elasticsearch/cluster/coordination/RareClusterStateIT.java

```diff
@@ -146,7 +146,9 @@ public void onFailure(String source, Exception e) {
     private <Req extends ActionRequest, Res extends ActionResponse> ActionFuture<Res> executeAndCancelCommittedPublication(
             ActionRequestBuilder<Req, Res> req) throws Exception {
         ActionFuture<Res> future = req.execute();
-        assertBusy(() -> assertTrue(((Coordinator)internalCluster().getMasterNodeInstance(Discovery.class)).cancelCommittedPublication()));
+        assertBusy(
```

Comment:
I still opted to increase this timeout. I ran ~10k iterations of the test with my changes. Before my changes, I'd see this wait take ~16s worst case which was now down to ~3s worst case, but that was on a very quiet system with just this test running -> 3s on my box will randomly exceed 10s on CI imo.